libcloud/compute/drivers/nttcis.py

```diff
@@ -4649,6 +4649,251 @@ def ex_create_node_uncustomized(self,
 
         return new_node
 
+    # DRS methods
+    def ex_create_consistency_group(self, name, journal_size_gb,
+                                    source_server_id, target_server_id,
+                                    description=None):
+        """
+        Create a consistency group
+
+        :param name: Name of consistency group
+        :type name: ``str``
+        :param journal_size_gb: Journal size in GB
+        :type journal_size_gb: ``str``
+        :param source_server_id: Id of the server to copy
+        :type source_server_id: ``str``
+        :param target_server_id: Id of the server to receive the copy
+        :type: ``str``
+        :param description: (Optional) Description of consistency group
+        :type: ``str``
+        :return: :class: `NttCisConsistenccyGroup`
+        """
+
+        consistency_group_elm = ET.Element('createConsistencyGroup',
+                                           {'xmlns': TYPES_URN})
+        ET.SubElement(consistency_group_elm, "name").text = name
+        if description is not None:
+            ET.SubElement(
+                consistency_group_elm, "description").text = description
+        ET.SubElement(
+            consistency_group_elm, "journalSizeGb").text = journal_size_gb
+        server_pair = ET.SubElement(consistency_group_elm, "serverPair")
+        ET.SubElement(
+            server_pair, "sourceServerId").text = source_server_id
+        ET.SubElement(
+            server_pair, "targetServerId").text = target_server_id
+        response = self.connection.request_with_orgId_api_2(
+            "consistencyGroup/createConsistencyGroup",
+            method="POST",
+            data=ET.tostring(consistency_group_elm)).object
+        response_code = findtext(response, 'responseCode', TYPES_URN)
+        return response_code in ['IN_PROGRESS', 'OK']
+
+    @get_params
+    def ex_list_consistency_groups(self, params={}):
+        """
+        Functions takes a named parameter that must be one of the following
+        :param params: A dictionary composed of one of the following keys
```

Comment:
Why not just use a ** ? It'll convert a dictionary to a keyword argument list automatically (or vice versa) see https://treyhunner.com/2018/04/keyword-arguments-in-python/

Reply:
The decorator handles the parameters.  The idea is to slowly move nttcis away form huge parameter lists and let the developer include the keywords much as one would with say with Django models.Models.